server.js

```diff
@@ -79,6 +79,20 @@ var tickFormats = {
   currencyK: value => `$${value && (value/1000 + 'K')}`,
 };
 
+var getTickStrategy = (request, columns) => {
```

Comment:
i think this would be better as a hash to be consistent with the `tickFormats`. 

something like:
```
var strategies = {
     period: (columns) => (axis) => {
       var columnIndicies = _.filter(_.range(columns.length), (i) => i % 2 === 0);
       return axis.tickValues(_.map(columnIndicies, (i) => columns[i]));
     },
     constant: () => (axis) => {
       return axis.ticks(5);
     }
   };